### PR TITLE
1821-cant-rename-stylesvariables-after-changing-the-parent-name-of-a-token-group

### PR DESCRIPTION
**Pull request by robinhoodie0823**
https://github.com/tokens-studio/figma-plugin/assets/103296157/262ddee1-7d21-4cf5-b324-04f9facd069e

### CODE REVIEW

File: src/app/components/DownshiftInput/mentions.css [hunk 1]
```diff
@@ -69,7 +69,7 @@
   border: 1px solid var(--colors-borderDefault);
   border-radius: var(--radii-medium);
   box-shadow: var(--shadows-contextMenu);
-  background: var(--colors-bgDefault);
+  background: var(--colors-contextMenuBg);
```
Comment (six7) [inline src/app/components/DownshiftInput/mentions.css:72]: this seems like a mistake, we only recently changed this

File: src/app/store/models/reducers/tokenState/renameVariableNamesToThemes.ts [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
       Object.entries(updatedTokens).forEach(([key]) => {
-        if (key === oldName) {
-          updatedTokens[newName] = updatedTokens[oldName];
-          delete updatedTokens[oldName];
+        if (oldNameMapNewName[key]) {
+          updatedTokens[oldNameMapNewName[key]] = updatedTokens[key];
+          delete updatedTokens[key];
         }
       });
     }
```
Comment (six7) [inline src/app/store/models/reducers/tokenState/renameVariableNamesToThemes.ts:18]: lets rename this variable to something more understandable: `oldToNewNameMap` for example

**Review by six7 — CHANGES_REQUESTED** [review]

File: src/app/components/DownshiftInput/mentions.css (hunk 1 above)
Comment (robinhoodie0823) [reply src/app/components/DownshiftInput/mentions.css]: Reverted.

File: src/app/store/models/reducers/tokenState/renameVariableNamesToThemes.ts (hunk 2 above)
Comment (robinhoodie0823) [reply src/app/store/models/reducers/tokenState/renameVariableNamesToThemes.ts]: Updated variable name to `oldToNewNameMap`

**Review by six7 — APPROVED** [review]

**Pull request merged**